### PR TITLE
Email activity timeline fix

### PR DESCRIPTION
**Pull request by mqueme**
Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #2313 #2231
| BC breaks? | 
| Deprecations? | 

**Note that all new features should have a related user and/or developer documentation PR in their respective repositories.** 

#### Description:
This PR includes failed emails in contact's timeline and also it fixes the messages shown for emails read and emails sent.
#### Steps to reproduce the bug:
1. have some emails with failed status and some with read status and some with only sent status
2. open the contact's timeline related to the emails sent. Notice that failed emails are treated as sent emails, and the descriptions under all of them are the same.

#### Steps to test this PR:
1. Apply this PR 
2. Notice now if an email is failed it says it has failed, and also if its been read it tell you correct info of when it was read.

### CODE REVIEW

File: app/bundles/EmailBundle/EventListener/LeadSubscriber.php [hunk 1]
```diff
@@ -102,13 +103,16 @@ protected function addEmailEvents(LeadTimelineEvent $event, $state)
                 } else {
                     $eventName = $label;
                 }
-
+                if ('failed' == $state) {
+                    $state = 'sent';
```
Comment (alanhartless) [inline app/bundles/EmailBundle/EventListener/LeadSubscriber.php]: I think we should indicate in the UI that it was failed and not sent (in some cases failed = failure to send at all)

**Comment by alanhartless** [comment]
I just saw this one too #2208 - maybe we can incorporate them both

File: app/bundles/EmailBundle/EventListener/LeadSubscriber.php (hunk 1 above)
Comment (mqueme) [inline app/bundles/EmailBundle/EventListener/LeadSubscriber.php]: this was changed to get the correct date column later in the code dateSent in the UI it reads as Falied

**Comment by alanhartless** [comment]
+1 works for me!

**Comment by dongilbert** [comment]
+1

**Pull request merged**